validator/sawtooth_validator/execution/context_manager.py

```diff
@@ -484,6 +526,40 @@ def delete_contexts(self, context_id_list):
             if c_id in self._contexts:
                 del self._contexts[c_id]
 
+    def delete(self, context_id, address_list):
+        """Delete the values associated with list of addresses, for a specific
+        context referenced by context_id.
+
+        Args:
+            context_id (str): the return value of create_context, referencing
+                a particular context.
+            address_list (list): a list of address strs
+
+        Returns:
+            (bool): True if the operation is successful, False if
+                the context_id doesn't reference a known context.
+
+        Raises:
+            AuthorizationException: Raised when an address in address_list is
+                not authorized either by not being in the inputs for the
+                txn associated with this context, or it is under a namespace
+                but the characters that are under the namespace are not valid
+                address characters.
```

Comment:
I think an authorization error will mislead whoever has to fix it. Two distinct errors would be ideal, but if we can only send back one error for both, the name should be rephrased to encompass both authorization and address syntax issues.

If it is outside the scope of this PR to fix this issue, a JIRA task should be made.